The type or namespace name 'UITest' does not exist ... when building in Visual Studio 2019

The SpecBind.CodedUI project fails to build in Visual Studio 2019 with the following error:
`error CS0234: The type or namespace name 'UITest' does not exist in the namespace 'Microsoft.VisualStudio.TestTools' (are you missing an assembly reference?)`

The `Coded UI Test` feature is installed in Visual Studio 2019.

Basically, the `CodedUI.targets` file does not support the condition where `VisualStudioVersion == 16.0` so the project references are not included during a build.

--- a/src/CodedUI.targets
+++ b/src/CodedUI.targets
@@ -53,6 +53,7 @@
       </ItemGroup>
     </When>
   </Choose>
+  <Import Project="$(VSToolsPath)\TeamTest\Microsoft.TestTools.targets" Condition="'$(VisualStudioVersion)'=='16.0' and Exists('$(VSToolsPath)\TeamTest\Microsoft.TestTools.targets')" />
   <ItemGroup>
     <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
       <HintPath>$(MSBuildThisFileDirectory)packages\CodedUIDependencies.1.0.0\lib\net46\Microsoft.VisualStudio.QualityTools.UnitTestFramework.dll</HintPath>

